### ISSUE TITLE
Dataset

### ISSUE THREAD

**Issue by Shahin-rmz** [saeedesmaili/iranian-businesses-data-reports#2]
Thanks for informative repo.
Is there any ways to have actual datasets of these reports?

**Comment by saeedesmaili** [comment]
I don't think so. These reports have been published by the companies themselves, and they have access to their own data sources. They're publishing some interesting and sometimes advertising insights based on those private data they have.

**Comment by Shahin-rmz** [comment]
Thanks for  the info.
BTW if there is any datasets and databases which are freely available for public, please update the repo so we can also benefit and play around with it.

**Comment by saeedesmaili** [comment]
Actually, that's another project that I'm working on. I'm gathering a collection of Iranian datasets that are publicly available. I'll publish it on my blog like this one.

**Comment by Shahin-rmz** [comment]
Nice.
have been subscribed.
although working on a project, if I see project due, would be happy to add new PR and colaborate.
